src/resources/Catalogs/CatalogInterfaces.ts

```diff
@@ -4,6 +4,8 @@ export interface CatalogsListOptions {
     pageSize?: number;
 }
 
+export type ConfigurationsListOptions = Omit<CatalogsListOptions, 'filter'>;
```

Comment:
We use `CatalogConfiguration` elsewhere in the file instead of `Configuration`, so I think we should go with `CatalogConfigurationsListOptions`